chromium_src/chrome/browser/ui/webui/settings/safety_hub_handler_unittest.cc

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "components/content_settings/core/common/content_settings_types.h"
+
+// clang-format off
+#define TOP_LEVEL_STORAGE_ACCESS                     \
+                           TOP_LEVEL_STORAGE_ACCESS, \
+      ContentSettingsType::BRAVE_FINGERPRINTING_V2,  \
+      ContentSettingsType::BRAVE_HTTPS_UPGRADE
```

Comment:
@bridiver Upstream is fixing all of their revocable types without a name to have a name. Should these 2 types be revocable?

Reply:
@goodov probably knows more about this

Reply:
> @bridiver Upstream is fixing all of their revocable types without a name to have a name. Should these 2 types be revocable?

these are not permissions, but shields settings. I don't think they should not be revocable.

Reply:
I haven't looked much into this but it looks like anything that has the initial value of CONTENT_SETTING_ASK is considered trackable and revocable (not sure if the missing names are a factor).

Reply:
@mkarolin would that make them fall into the new 60 day revocation or whatever it is? If so I don't think we'd want that like @goodov said

Reply:
some of these settings use `CONTENT_SETTING_ASK` as a huge hack to have three possible values instead of two